app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<h>Hello World</h>
+<main class="home">
+  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+  <section class="home__section">
+    <%= render 'svg/home-ill-cup' %>
```

Comment:
Wouldn't it be better to render this outside the `home__section` in its own section?